From: Philippe Coval <philippe.coval@osg.samsung.com>
Date: Fri, 25 Aug 2017 14:38:07 +0200
Subject: build: Install headers to system

make install will install headers to iotjs headers namespace.

This is needed for build developer packages for most Linux distros:
Tizen, or any debian based system like raspbian for pi0.

Bug: https://github.com/Samsung/iotjs/pull/1145
IoT.js-DCO-1.0-Signed-off-by: Philippe Coval philippe.coval@osg.samsung.com
Forwarded: https://github.com/Samsung/iotjs/pull/1145
Origin: https://github.com/Samsung/iotjs/commit/1253e15f55623d6bd420c36f5092fac9e4393bb7.patch
---
 cmake/iotjs.cmake | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/cmake/iotjs.cmake b/cmake/iotjs.cmake
index b39f3c1..ce54711 100644
--- a/cmake/iotjs.cmake
+++ b/cmake/iotjs.cmake
@@ -265,7 +265,13 @@ if("${BIN_INSTALL_DIR}" STREQUAL "")
   set(BIN_INSTALL_DIR "bin")
 endif()
 
+if("${INCLUDE_INSTALL_DIR}" STREQUAL "")
+  set(INCLUDE_INSTALL_DIR "include/iotjs")
+endif()
+
 install(TARGETS ${TARGET_LIB_IOTJS} DESTINATION ${LIB_INSTALL_DIR})
+file(GLOB IOTJS_HEADERS include/*.js src/*.h)
+install(FILES ${IOTJS_HEADERS} DESTINATION ${INCLUDE_INSTALL_DIR})
 
 if(NOT BUILD_LIB_ONLY)
 
